README.md

```diff
@@ -36,5 +36,5 @@ Example Usage
 ```sh
 # Download latest Kafka distribution(if necessary)
 tar zxvf kafka_LATEST_VERSION.tgz
-./kafka_consumer_group_exporter --consumer-group-command-path=./kafka_LATEST_VERSION/bin/kafka-consumer-groups.sh BOOTSTRAP_SERVERS
+./kafka_consumer_group_exporter --consumer-group-command-path=./kafka_LATEST_VERSION/bin/kafka-consumer-groups.sh BOOTSTRAP_SERVER
```

Comment:
I changed to singular because I thought the help text for `kafka-consumer-groups.sh` hints that it only takes a single server:
```sh
$ /Users/jens/Development/src/external/kafka_2.11-0.10.0.0/bin/kafka-consumer-groups.sh
List all consumer groups, describe a consumer group, or delete consumer group info.
Option                                 Description
------                                 -----------
--bootstrap-server <server to connect  REQUIRED (only when using new-
  to>                                    consumer): The server to connect to.
--command-config <command config       Property file containing configs to be
  property file>                         passed to Admin Client and Consumer.
--delete                               Pass in groups to delete topic
                                         partition offsets and ownership
                                         information over the entire consumer
                                         group. For instance --group g1 --
                                         group g2
                                       Pass in groups with a single topic to
                                         just delete the given topic's
                                         partition offsets and ownership
                                         information for the given consumer
                                         groups. For instance --group g1 --
                                         group g2 --topic t1
                                       Pass in just a topic to delete the
                                         given topic's partition offsets and
                                         ownership information for every
                                         consumer group. For instance --topic
                                         t1
                                       WARNING: Group deletion only works for
                                         old ZK-based consumer groups, and
                                         one has to use it carefully to only
                                         delete groups that are not active.
--describe                             Describe consumer group and list
                                         offset lag related to given group.
--group <consumer group>               The consumer group we wish to act on.
--list                                 List all consumer groups.
--new-consumer                         Use new consumer.
--topic <topic>                        The topic whose consumer group
                                         information should be deleted.
--zookeeper <urls>                     REQUIRED (unless new-consumer is
                                         used): The connection string for the
                                         zookeeper connection in the form
                                         host:port. Multiple URLS can be
                                         given to allow fail-over.
```
Am I wrong? Does the help text differ for later Kafka versions, perhaps?